src/Annotation/AnnotationCollection.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-code for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-code/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-code/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Form\Annotation;
+
+use ArrayObject;
+
+use function get_class;
+
+/**
+ * @extends ArrayObject<array-key, object>
+ */
+class AnnotationCollection extends ArrayObject
```

Comment:
can we mark this final? imho no need to provide abstraction support here

Reply:
Sure, done, thanks